tests/test_dataUnit.py

```diff
@@ -33,7 +34,8 @@ class DataUnitRegistryTestCase(lsst.utils.tests.TestCase):
     """
     def setUp(self):
         self.testDir = os.path.dirname(__file__)
-        self.schemaFile = os.path.join(self.testDir, "../config/registry/default_schema.yaml")
+        self.schemaFile = os.path.join(lsst.utils.getPackageDir("daf_butler"), "config",
+                                       SchemaConfig.defaultConfigFile)
         self.config = SchemaConfig(self.schemaFile)
```

Comment:
Do we still need to specify the `schemaFile`? Or does calling `SchemaConfig()` load the default file?

Reply:
At the moment, the only defaulting is in `ButlerConfig`. I can see how each subclass doing its own defaulting and getting the search path from a shared routine might be helpful (and more obvious).